src/Components/Buttons.jsx

```diff
@@ -89,11 +90,12 @@ export function ControlButton({
   setIsDialogDisplayed,
   icon,
   placement='left',
+  size = 'medium',
   dialog,
 }) {
   assertDefined(title, isDialogDisplayed, setIsDialogDisplayed, icon, dialog)
   const toggleIsDialogDisplayed = () => setIsDialogDisplayed(!isDialogDisplayed)
-  const classes = useStyles(useTheme())
+  const classes = useStyles(size === 'small' ? {buttonWidth: '40px'} : {buttonWidth: '50px'})
```

Comment:
Shouldn't this be in a @media selector down in the styles defs?

Reply:
Since we are styling the button explicitly, MUI small prop does not work. 
I added a size prop to all of the button components, the default is medium for all of the buttons. 
If the size is small it passes a bottonWidth Prop to the styles.
<img width="432" alt="image" src="https://user-images.githubusercontent.com/3433606/163228882-9f9772cf-57a8-41aa-85c7-1ed5cf5e4e2b.png">